dependency/vault_common.go

```diff
@@ -92,14 +92,19 @@ func renewSecret(clients *ClientSet, d renewer) error {
 		case err := <-renewer.DoneCh():
 			if err != nil {
 				log.Printf("[WARN] %s: failed to renew: %s", d, err)
+				//d.recordCounter("status", "stopped")
```

Comment:
In the original PR #1378, there were metrics updated each time a token was renewed.
I aimed to apply the same logic but couldn't due as adding metrics in the dependency package now induces import cycling.
I didn't want to re-organise the whole codebase so I just commented out these parts.

I am unsure of how I should proceed though. Should I:
- remove the comments and it will be the job of someone who wants metrics on token renewal?
- update the commented code (in order to at least make it similar to other counters), and add a bit of context in the comments?

Reply:
I removed the comments for now, as we could always implement metrics later.
The main focus of this PR was to at least implement metrics related to the templating.